packages/software-terms/softwareTerms.txt

```diff
@@ -92,16 +98,20 @@ btoa
 bubble
 bugfix
 bugfixes
+bundler
+bundler's
+bundlers
+bundling
 c
 cache
 calc
 calcs
 callbackfn
 callee
-Capitan
```

Comment:
I added this on from the macOS "El Capitan"

Reply:
Ahh, that make sense now.